src/tensor_computes/LBMComputeVelocity.C

```diff
@@ -0,0 +1,86 @@
+/**********************************************************************/
+/*                    DO NOT MODIFY THIS HEADER                       */
+/*             Swift, a Fourier spectral solver for MOOSE             */
+/*                                                                    */
+/*            Copyright 2024 Battelle Energy Alliance, LLC            */
+/*                        ALL RIGHTS RESERVED                         */
+/**********************************************************************/
+
+#include "LBMComputeVelocity.h"
+#include "LatticeBoltzmannProblem.h"
+#include "LatticeBoltzmannStencilBase.h"
+
+using namespace torch::indexing;
+
+registerMooseObject("SwiftApp", LBMComputeVelocity);
+
+InputParameters
+LBMComputeVelocity::validParams()
+{
+  InputParameters params = LatticeBoltzmannOperator::validParams();
+  params.addRequiredParam<TensorInputBufferName>("f", "Distribution function");
+  params.addRequiredParam<TensorInputBufferName>("rho", "Density");
+  params.addParam<TensorInputBufferName>("forces", "forces", "Force tensor");
+  params.addParam<bool>("enable_forces", false, "Whether to enable forces or no");
+  params.addParam<bool>("add_body_force", false, "Whether to enable forces or no");
+  params.addParam<std::string>("body_force_x", "0.0", "Body force to be added in x-dir");
```

Comment:
Note to self, we eventually will need a derived string class for those, but it is not yet possible for apps to register those